src/seedu/addressbook/common/UtilsTest.java

```diff
@@ -9,21 +9,21 @@
 public class UtilsTest {
 
 	@Test
-	public void isAnyNullTest() {
+	public void isAnyNullTest_noNull_returnFalse() {
 		String[] listOfString = {"hi","hello","heehee"};
 		boolean result = Utils.isAnyNull(listOfString);
 		assertEquals(false, result);
 	}
 	
 	@Test
-	public void isAnyNullTest2(){
+	public void isAnyNullTest_nullInObject_returnTrue(){
```

Comment:
much clearer naming:) don't need 'Test' in 'isAnyNullTest' though since we already have `@Test`